### PR TITLE
[DPE-1438] Logs Integration 

### PR DESCRIPTION
**Pull request by deusebio**

### CODE REVIEW

**Review by deusebio — COMMENTED** [review]
LGTM. I'm about to test this today

@zmraul

**Comment by deusebio** [comment]
Tests were successful. I found few issues (minors to be addressed in https://warthogs.atlassian.net/browse/DPE-1432) or by the COS team (https://github.com/canonical/grafana-agent-k8s-operator/issues/158).

File: src/charm.py [hunk 1]
```diff
@@ -93,6 +93,9 @@ def __init__(self, *args):
                 # See https://github.com/canonical/charmed-kafka-snap for details
                 {"path": "/metrics", "port": f"{JMX_EXPORTER_PORT}"},
             ],
+            metrics_rules_dir="./src/alert_rules/prometheus",
+            logs_rules_dir="./src/alert_rules/loki",
+            log_slots=["charmed-kafka:logs"],
```
Comment (marcoppenheimer) [inline src/charm.py:98]: **nitpick/todo:** Could these be `literals`?

Or better, have `log_slots` added as class vars to `KafkaSnap`, and the others go to `literals.py`?

**Review by marcoppenheimer — APPROVED** [review]
Added a minor nit, otherwise lgtm

File: src/charm.py (hunk 1 above)
Comment (zmraul) [reply src/charm.py:98]: Will do in a following PR

**Pull request merged**